### DISCUSSION TITLE
Where is legened-state-optimized?

### DISCUSSION THREAD

**Discussion by HamedFathi** [LegendApp/legend-state#133] [General]
Hi, you share the below picture

[](https://www.legendapp.com/img/dev/state/times.png)

I want to know what is the difference between optimized and non-optimized versions?

 I did not find the optimized version in the krausest/js-framework-benchmark too.

**Comment by jmeistrich** [comment]
I submitted a PR for it but it hasn't been accepted yet: https://github.com/krausest/js-framework-benchmark/pull/1269

The optimized version basically just adds the `optimized` prop to the `For` component.

**Comment by jmeistrich** [comment]
Update: It's in the official results now. The optimized version is in the non-keyed table because it does not re-render the whole list so it doesn't match the benchmark's keyed requirements.
